### PR TITLE
Auto mirroring, simulation, implemented; WPILib version to 2023.4.3

### PR DESCRIPTION
**Pull request by MGuron**
TODO: Increase speed of pathplanner autos

### CODE REVIEW

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -84,11 +76,6 @@ public RobotContainer() {
     m_armSubsystem.setDefaultCommand(
         new RunCommand(() -> m_armSubsystem.proceedToArmPosition(), m_armSubsystem));
 
-    // if (DriverStation.getAlliance() == Alliance.Red) {
-    //   twoHighBump = "REDTwoHighAuto";
-    // } else {
-    //   twoHighBump = "BLUETwoHighAuto";
-    // }
     twoHighBump = "BLUETwoHighAuto";
```
Comment (MGuron) [inline src/main/java/frc/robot/RobotContainer.java:79]: This path is the only one placed into an instance variable for it's autonomous command, does this one need to be different than all other paths?

File: src/main/deploy/pathplanner/TestMirroring.path
Comment (MGuron) [inline src/main/deploy/pathplanner/TestMirroring.path:1]: Does this path need to stay in the repo?

File: src/main/java/frc/robot/RobotContainer.java (hunk 1 above)
Comment (MGuron) [reply src/main/java/frc/robot/RobotContainer.java:79]: Also, the BLUETwoHighAuto can be renamed to not include the alliance color, REDTwoHighAuto can be deleted

File: src/main/java/frc/robot/commands/autonomous/BasePathAuto.java
Comment (MGuron) [inline src/main/java/frc/robot/commands/autonomous/BasePathAuto.java:1]: ConeHighBalance should inherit from this; autoTrajectory could then be deleted

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -62,8 +55,7 @@ public class RobotContainer {
   private final LEDSubsystem m_LEDSubsystem = new LEDSubsystem();
   private boolean isAuto;
   private String twoHighBump;
-  // public boolean isRedAlliance = (DriverStation.getAlliance() == Alliance.Red);
-  public boolean isRedAlliance = false;
+  public boolean isRedAlliance = true;
```
Comment (sfwasleski) [inline src/main/java/frc/robot/RobotContainer.java:58]: The name isRedAlliance is not what this boolean really represents when you get to the path planner classes. Could we rename this and other usages to useAllianceColor (what path planner calls it) or enableMirroring (also accurate)?

**Review by isaiahdrago — APPROVED** [review]

**Pull request merged**